src/preset_cli/cli/superset/sync/dbt/command.py

```diff
@@ -338,7 +348,44 @@ def get_job(
     raise ValueError(f"Job {job_id} not available")
 
 
-def process_sl_metrics(
+def get_sl_metric(
+    metric: Dict[str, Any],
+    model_map: Dict[ModelKey, ModelSchema],
+    dialect: MFSQLEngine,
+) -> Optional[MFMetricWithSQLSchema]:
+    """
+    Compute a SL metric using the ``mf`` CLI.
+    """
+    mf_metric_schema = MFMetricWithSQLSchema()
+
+    command = ["mf", "query", "--explain", "--metrics", metric["name"]]
+    try:
+        result = subprocess.run(command, capture_output=True, text=True, check=True)
```

Comment:
When there are multiple metrics defined, this process might take some time which would be invisible to the user. It might be interesting to add a logger here so the user can follow up as the metrics are getting evaluated. For example:
``` python
logger.info("Using `mf` command to retrieve SQL syntax for metric %s", metric["name"])
```